ocs_ci/ocs/ui/page_objects/external_storage_systems.py

```
@@ -254,21 +248,50 @@ def connect_scale_filesystem(self, scale_name, filesystem_name):
+        if self.get_elements(self.external_systems["modal_error_alert"]):
+            error_msg = self.get_element_text(
+                self.external_systems["modal_error_alert"]
+            )
+            logger.warning(
+                f"Filesystem creation returned an error modal: {error_msg}; dismissing modal"
+            )
+            if self.get_elements(self.external_systems["modal_cancel_button"]):
+                self.do_click(self.external_systems["modal_cancel_button"])
+            raise CommandFailed(
+                f"Failed to add scale filesystem '{filesystem_name}': {error_msg}"
+            )
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)